src/main/java/model/Car.java

```diff
@@ -0,0 +1,125 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Arrays.*;
+import static java.util.stream.Collectors.toList;
+
+public class Car {
+    private int score;
+
+    private final String name;
+
+    public static final int START_SCORE = 1;
+
+    public Car(String name, int score) {
+        this.name = name;
+        this.score = score;
+    }
+
+    public static List<Car> getWinner(List<Car> carList) {
+        int maxScore = getMaxScore(carList);
+
+        List<Car> maxCars = carList.stream()
+            .filter(car -> car.isMaxScoreCar(maxScore))
+            .collect(toList());
+
+        return maxCars;
+    }
+
+    public static List<String> getWinnerName(List<Car> carList) {
+        return getWinner(carList).stream()
+            .map(car -> car.name)
+            .collect(toList());
+    }
+
+    private boolean isMaxScoreCar(int maxScore) {
+        return this.score == maxScore;
+    }
+
+    private static int getMaxScore(List<Car> carList) {
+        return carList.stream()
+            .map(car -> car.score)
+            .max(Integer::compareTo)
+            .orElse(START_SCORE);
+    }
+
+    public static List<Car> getCar(String[] carNameList) {
+        return stream(carNameList)
+            .map(carName -> {
+                return new Car(carName, START_SCORE);
+            })
+            .collect(toList());
+    }
+
+    public static void validate(String str) {
+        String[] carNameList = str.split(",");
+        HashSet<String> checker = new HashSet<>();
+
+        for (String carName : carNameList) {
+            validateCarNameLength(carName);
+            checker.add(carName);
+        }
+
+        if (checker.size() != carNameList.length) {
+            throw new IllegalArgumentException("동일한 차량 이름은 불가능합니다.");
+        }
+    }
+
+    private static void validateCarNameLength(String str) {
+        if (str.length() > 5) {
+            throw new IllegalArgumentException("차량 이름은 5자를 넘어서는 안됩니다.");
+        }
+    }
+
+
+    public static Car updateCarScore(Car car, int num) {
+        if (num > 3){
+            car.score++;
+        }
+
+        return new Car(car.name, car.score);
+    }
```

Comment:
객체란 자신만의 고유한 상태와 행위를 가지는 것을 의미합니다. 
Car 객체는 자신의 전진하게되면 자신의 score 상태를 증가시킬거에요. 

아래와 같이 변경해보면 어떨까요? 🙂

```suggestion
    public void updateScore(int num) {
        if (num > 3){
            this.score++;
        }
    }
```

Reply:
해당 부분 관련해서 수정 사항이 많아 추후 리팩토링 시 다시 진행하겠습니다..🥲